crypto/src/test/java/org/cryptimeleon/incentive/crypto/IncentiveSystemTest.java

```diff
@@ -162,7 +163,7 @@ public void fullProtocolTestRun() {
                 i -> finalUpdatedToken.getPoints().get(i).asInteger().subtract(newPointsAmount3.get(i)),
                 newPointsAmount3.length()
         );
-        var spendDeductTestZkp = SpendHelper.generateSimpleTestSpendDeductZkp(incSys.pp, promotionParameters, pkp.getPk(), spendAmount);
+        SpendDeductTree spendDeductTestZkp = SpendHelper.generateSimpleTestSpendDeductTree(incSys.pp, promotionParameters, pkp.getPk(), spendAmount);
```

Comment:
change for readability?

Reply:
Actually not sure, but works perfectly ;)